mdpow/fep.py

```diff
@@ -725,10 +728,13 @@ def convert_edr(self):
             edr_files = [self.dgdl_edr(self.wdir(component, l)) for l in lambdas]
             tpr_files = [self.dgdl_tpr(self.wdir(component, l)) for l in lambdas]
             for tpr, edr in zip(tpr_files, edr_files):
-                deffnm = os.path.splitext(edr)[0]
-                xvgfile = deffnm + ".xvg"  # hack
-                logger.info("  {0} --> {1}".format(edr, xvgfile))
-                gromacs.g_energy(s=tpr, f=edr, odh=xvgfile)
+                dirct = os.path.abspath(os.path.dirname(edr[0]))
+                total_edr = os.path.join(dirct, 'total.edr')
```

Comment:
Where does `total.edr` come from? Why is there a hard-coded name??

At least make the name configurable as a kwarg
```
def dgdl_tpr(self, *args, **kwargs):
   total_edr_name = kwargs.get("total_edr_name", "total.edr")
```